CONTRIBUTING.md

```diff
@@ -426,6 +432,14 @@ To become a Code Owner, open a PR with the following changes:
 
 Be sure to tag the existing Code Owners, if any, within the PR to ensure they receive a notification.
 
+### Emeritus roles
+
+Contributors that are stepping down or reducing their role in the project are encouraged to move to [emeritus](https://github.com/open-telemetry/community/blob/main/guides/contributor/membership.md#emeritus-maintainerapprovertriager). 
+
+In the event that a contributor steps down without prior notice, the maintainers will attempt to contact the contributor via both Github and the CNCF Slack to confirm their status. After two weeks, if the contributor is an approver or maintainer, they may be removed from the Github review auto-assignment. If the contributor does not respond within a period of one month, they may be moved to emeritus status at the discretion of the maintainers.
```

Comment:
Do we have clear expectations for a contributor's involvement?
What a contributor with a specific role should do in case there is planned "time off" for any kind of reasons?
Some examples can be when people go on long time off (1+ month), time off for medical reasons, paternity leaves etc.

Reply:
> Not sure if that's on-topic but reading the "timeline" based steps I wonder if we have clear expectations for a contributor's involvement?

That's fair, yes, I think we should mention something related to this. The timeline is meant to apply in case there is no prior notice, maybe defining what that prior notice is would be enough? (e.g. a message/status on Slack is what people typically do, we could mention that explicitly).

> Some examples can be when people go on long time off (1+ month), time off for medical reasons, paternity leaves etc.

Personally I am fine with making the second part (moving to emeritus) longer, I think the more important bit is that the day-to-day activities continue, so I am fine with a short timeline for "remove from autoassignment" and a longer one for "move to emeritus"

Reply:
PTAL at 20fd76856b1047b019407f903fdf35bc61716230

Reply:
Marking as solved as we discussed this on the SIG meeting